internal/pkg/deploy/cloudformation/stack/testdata/workloads/backend/simple-template-without-port-config.yml

```diff
@@ -0,0 +1,387 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is it possible not to add a new `testdata` template? It's a real pain maintaining all of them - for example, when all I've done is adding a new `Parameters` and I have to modify > 10 `testdata` templates just to pass the integration test. 

Maybe we can instead modify an existing `testdata` to test the case of "without-port-config`?